add emerald vault dao's upgrade celebration instance

A60AB5450353F40E opened the pull request:
is this ok? all nfts are supposed to have the same pic

A60AB5450353F40E committed:
add emerald vault dao's upgrade celebration instance

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

bitjson commented:
Hey @A60AB5450353F40E, thanks for the PR, and congrats on the launch!

This is looking great so far – a few things:

### Icon

I verified that your `icon` is a 400px by 400px PNG. Could you update the URI to use the IPFS protocol? `ipfs://QmcNL1KcVmiDtwJe8WokrnzYeoHirsz1sNxNojncsxyb2p` – Clients can use the `pat.mn` gateway, but we don't want the underlying registry to be vulnerable to changes by any other domains.)

### Description

Can you provide some evidence for this part of your `description`?

> Starting May 15th 2024, whomever holds the NFT will have the option of burning the NFT to get 0.101 BCH back from the DAO.
>The additional 0.001 BCH comes from DAO's rewards pool which was seeded by the creator.

Maybe the code of the contract and some on-chain tests demonstrating the withdrawals will work as claimed?

### `web` URI

[Here's our current requirement](https://otr.cash/docs/list#inclusion-criteria) for web presence:

> **An active web presence** – At a minimum, all entries must have an active `web` URI offering information about the token; generally, this should be a dedicated project website at a unique domain name owned by the project or issuing organization. The `web` URI may use IPFS (or other content-addressing protocols), but such entries must still demonstrate ongoing activity by other means, e.g. an active `forum`, `chat`, `support`, or other [optional URI identifiers](https://cashtokens.org/docs/bcmr/chip/#optional-uri-identifiers). Entries with consistently non-responsive `web` URIs may be moved to an archival registry or removed from OpenTokenRegistry.

In `web`, could you link to a dedicated project website of some sort?

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E commented:
>Icon

Fixed

>Description

Sources available [here](https://gitlab.com/0353F40E/emerald-dao/-/tree/main/contracts/v2-1-0), and BitauthIDE template of compiled code is [available](https://gitlab.com/0353F40E/emerald-dao/-/blob/main/instances/v2-1-0/180f0db4465c2af5ef9363f46bacde732fa6ffb3bfe65844452078085b2e7c93/emerald_dao_v2.1.0.bitauth-template.json), direct import link [here](bitauth-20u91jodn-mainnet-pat.vercel.app/import-gist/53dfd0b9caf42a5ff63df2f5bc400a04).

Should I make an instance on testnet4 and make some TXes?

>web

Made a simple page and added it to the bcmr, how's this: https://0353f40e.gitlab.io/emerald-dao/
Edit: Kallisti registered a domain for this, updated with https://emerald-dao.cash/ (still hosted at my gitlab site)

>chat

Added this too, there's a Tg group now (created by Fouad): https://t.me/emeralddao

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E commented:
## Chipnet Testing

DAO instance contracts: 

- Blueprint contract address: https://chipnet.imaginary.cash/address/bchtest:prqd82cxtdfgss8x0ffe7h6qwyg9c9vjtcjgdjvszy (sha256 of static "tail" part of redeem script bytecode: `c2ac9207655ecca85dcdc3c0fae33233a719afcf40a92a41e02911bce41ede1c`)
- Vault contract address: https://chipnet.imaginary.cash/address/bchtest:pzdh0j90jl7f4xpjmr5x5676czug7lsc4cwtqk736l (sha256 of static "tail" part of redeem script bytecode: `f1e36e373a10be68444155b0bd4ef81f8d1973693534c129cb19f451a9a94fe7`)
- Safebox contract address: https://chipnet.imaginary.cash/address/bchtest:pz4s5whwpzrxrqa065n8jva8pzdv44hx9qepr7u0st (sha256 of static "tail" part of redeem script bytecode: `09b65c077a62fe065580ee949ab77a695ff82806915fc1b3237f0c34e96f7f26`)

DAO configuration (from configured [BitauthIDE template](https://bitauth-20u91jodn-mainnet-pat.vercel.app/import-template/eJztPYty3DaSv8Kb3Yrj7EjmY566rK9kSYl1sTQ-SXaci11TIAlKXM-Qs0OOHrflf79uPEiABDkc2Um8d-tKRRIJNBqNfqMB_qP35yy4oUvSO-jd5PkqO3j2LA7pvh_nZJPf7Afp8hn-QpM8Dkgep8leTperBcnp3q29z_vu_y1Lk16_F9IsWMcrbAXgTpZ0TRahdXw4s-LMIlYWQ0dqvYjzII0T64hkN-ylBGjd3cTBjUWCHFpjhyi-pyGMt15aIV2lWZxbGbmNk-vMuiWbRW7dxfmNlbIBycIK1uldGG2SkIbWmt6RdZhZDEG6_z45IQBaQEnXVpwAWBgJgHEoMEWGzF28WFg-BWCUsLfEyuMl3VukwUeAm5GI-um99ebq3cwiSQivlyQPbrDlR_oQwKDW-Q9X7D2M-maVJha9X8XrByuN2CAFtL61yeiaD3hDbil7yydj5anlb9YJe1QFiy_TFeUvJT4MlSCgWcYei4kCwi-OXlp_sW7SRYgocrL0LaAAtFqSB4BOV7VhYqBhFCcAYPHAlyJIFwsK5PIXdB9WOiFLWlniW3ff2bfhHbJKHtOsd_CPHs4Qf-qcUQJ4g-_7vVuyjgmALvrMAZu2fkWbfm-1jm-BeY4p-wkNX5P8Bposn9nPEJ38YYU9XoY_QfNPfd41i69vgP3mSZTPgcWXcb4EtOsjnlOK3AQUv6YJzBWYNEjXa6AEcPN1QvLNmsIyImmBwrCat3Sd4QLCYp8en_StgCTITWu6TG8BELL5er8yjwZkCtQPw3ANC3tMclKbAMgkvU7XD_M4_IOxVzExo_4JkA9JWseTaYgky0kSIIZJFF9v1mwtrRVZw1gwbKaMC-2rTANw57znfEnu50IsxLsGJmrsU6D_MwGuzyXhlfai7TxJlzGonvktWWxox7HMfbuPibI_Ry2y43hlv-1jMfU6DxZpRncZqtatYSRkxyzO5vSeBhtQx_OItg5hbN8kH7KxHywZd3I-7wK-0qNpAGXCwszAzyVBpbme-wArioOYrFsVWGcYWyjIUE43-WqTz_2HnAZp2H2q1X7bKLqEZ-S6E3zZdBtIgAP0hjmvNv4Wnd_UZdsQrSqyDn67HjNyAMhyvNws53ECnoPZmLSufr3_dintKj5NXbaPEG6yfL6IwSR1hK902A4dcJjDu_QOFX_HAfQ-xjGYnRFMYjA24s2TzAIGWsQBej6F23cDi7GHwhCCRZJOITT1QUeDp5HkfeZq3YBDW5pQ2XlN_76J19A1Av-K9xNcZV29A2_wCh22oleQorm10gR8LBxIMt7pMRsiitdZbnkKBIvLa2ah94hGWwpupsCG0aElwwiArimgQhG3JRIIzSW6A4yAMMpDmrC_gwWJlwwJP90k-QN3iYMbklzjOPhigJ42G_9JHQEYf4a2u5wcILwkIRUk7sux0KFgYKk62oo8AGhE5CZeZnQRIQHeJzk4zfgMfWMkuOKdYGeFMCA8NFQdBLn6VS9Bqo4tqkZr1-ZiugYXk_FfFtAERk6rzkmIHMoe8Z-azjbLXW9i4yDNctNzRtUWDf5Jz7HZv15XFWbuoOsdc5uKF9Zzay3qPgZMZDIYeHVgBtdHtHUaZ9Ngknv2_XhEps5gOBlFnhd6QTgeeTRwXEqciTsl4cQbBCMKbwZDx3P8yYQEPaFSKj6r7qrikvZLHcBCuBiCo7csVEWRvhTBWl1-S9aTKrIw1FLxtLKO2aHqDeXCtLsLBU2o4_iB7Q-GNPQnwTggAZn4Iy8ahLZLAzJypqEfuJ7NaGLwC3pPXpLV6kEP8c8gynRt17M2q-s16oSQPPzbk16LD6DJ3z5XIUwUzcsgFUFBqsaleFG0qGkwDFzvc5qEWcWgIbUbVye_37YimvcBhKbU9wgdOoE9mFJnNA3I1HYiMnKng4HjjH1n7AYu9YewBpE7JOOI-N408IbToT_0pv54Cw3yNUkyzGsAQy6YWq3PrIG1ykmCRivB4GTiBO1O7-BXiM4TQboXxWR_7WWLNO996PeQs2Dd86uy-0tgAYDpjwF5mARMJPC9iJCxCxMcwKRc6gbOGCbuDAYgfiMnmtrAakCcwTSwhw4BgjFfRQI_BaEGstufxIASsxpe_-hxKlXcXlxDpg8vSZ5mYFh6B6BhP_XbISDhuMZauTCjOozBlEkb0C79SBnV5NJ_meknUc6BrogfL-IcwYLGxhwVvFWTGD0m9GiGuhElXYGyBKudFIJcn52ZPr0RsZm6Htk9Q5cPaAjTzTqgs7Z1KvmnAoGrLw5HlzpQXnPNxRMSeIMmOOO_veZWWzz41F1uZLIkBO0gsml8pGZRAj1QD7EUFI0TKJJjLeqjrhvtexsVsuNGoBaiKIgGNPImke0PiRsMR5PRcDByhsTxHd8LXW8yHoWBQ0ch9QZ-ZHvDsedNqxZBiZIBvmcPhrbrgmkNwSZGYAI8fxQFU3fggBWn0yEYRTsEBrbhMQkd3x6N6cCdgCYbjqNoMB0RfzDww-EUoNhDd0zJJByQYDD2wNT69tQmNh0MgmAwGg9BA06jceSPQ-q60cChdBI4Uyei0_EQ_vNHSLsvv6zghyL5d15RtmZiNavrtxOWmPnMWOgBYBhKhcto3aHjzrPDMk-daVnqjtYKx7BE2rPUXyI3PAcvgYQh8qJU7waHWB9lw9O1v6GdsCeODwbPHYa2PRyOp04IzoY9HQwHwN90DBwzCQM_DMbwwAuHwNckCEdB4E8Ho2AcjU12om_ERlGELI_pBzeKHHz6fZF0PltXuuDRDkejr84AVdB0XGYvAMtOC7JyVx_B2BYWQOdg4X1pPHuE-zYUd5uEsy0a10VmizQowSBrUed21fmorwZwg-d6X8VqOJ3cAbmngrtAbC9CEN_kLf2u00rShJrmBKZlNCV28a_DHHHdZTDZ4Mc5ANp1x-OvYoo4pQ-fjMyP9mDOcylGQdjC76p9M4D8F7f_i9v_EG7vsmyKZahNaDpxvJFXFRqu2utCchiy7ICsFlil6eLxhgIe7yY27ldosg2E26ZpvjQRH6-HvkaCfj5DTxyVoaVAI5HCNbkji5YsVGuxA4aSENBFo8HUjhTF0lZiAH2m3jhw6dC3J-OJC6Ei6P4JHfkRkBTzWNHYg7AxgAeDkTf1aBQNXTKEWM8f2KEdORNDButnMRcrWqdLra6F1dFsiUNKStS5BUnPE8Zlvri_c5wivHS3LSgYIcYuccYendCJP3Lc0dDzbKAVcUfeAEJy4jlhZE8idzoZjYgNBBvaZDoZR9PhKIgA94z-fUOTgJ5vlj6Gll1DF9WMGUMYGQq14T92RtEwCELPicJwBKJih-EYQv_AHoSD0AudiU1G48AbhdF47BLfGdEQJua7k8F4PPRCI_47pOi2CoKjZIN2ipT-aMvmMkVgmo8zHQ8e76h8BfOq6Q7DPCdKBk8qr2BBSTJfpYvFBlmwmwOrS30FxGOTEZ_DoruyYiWMFVtfu0-9da7NEh6CZI_tICATSuiUDnwyCPwgcKhDA-CCkTsejYfu1AN1PR7YJJqEkRe6BBz7cTQYT4kht9JvJ-0n43zZDh72mkOsHtxQc1mGyX0oKFaQogaqzQKMneF4PBjY9pfUSp43_qpVUpN70qSV5Hw6rBt31IqN1p2Xsc4IjetaHarVLfz8Be28GWTeZa5DHY-9wf8pNgFl7w4UzV5ZWW37_wvIuRneb8wFj18vssTKFqYglaUbg2cFrnAIKwSrB0o1BJfZjsaj0cB2iT3x6NiLvOGQRONo5E_90cCbTP3Ad4bUd1yy49KFtj3eVcKNZP5sMW9avLa1_WcX-OHnuapfNQM1yf7_U5-OlXshuLLYi21CVDepVyTLsBbq17bN7A960WMdUL_E_dkzVr-SyWj5goaULiWO75PvFZf7-fsEmmPpi3iATuFrTMYAaaHpn7_9vljK59b3znNr9np--frV6RX-cnwxe219P2APD4-Pn3JofHDWxTg0B8aaYnICJlBspV_KMBWbqtU-2LAMYvf1bWqyyGkIAHvS78zU3VlZzlJUmRbEkzu1bSvA2jRRX7Dpl6O8OxwpxHenX5Sg5r39TnQzp-nSoqoupP7muk5KLcu6JX_YsJPXYY-jXJ1uaOoLhmlKrMvlnRgR2b9TplMOLNvew5LchGCRr-3s4UkyEPmyodAH1rcA6Wml_a_QgSP6wdBBjFntBH1EYA-_u3uCPRCYVwLT1qwgEa6VIRMIlKjq0NpatcTgbRnGkvjdxtWJ_3PR1kj4Szl1IMlPnCQ1MrKGwO5gqgiXG9bkCMetA82xYjhOsPDYQk3NSpCtFGuiLWZFDPChRX7DDaFOcjHj4vSVOeEGlK4cTzM0UsiCOle2xJOUSbpezyv6rwZJVrHI9Czoavt-wNX10eEVauYSaFk6-VybU3NOyVAKUZmW_lKdTm0u5TTIYiHKTrMmBCucXnfbtpKibQFqbXX2jCNxDo4sVjdEPf3at9AnAoeIcxSW8W6S8hHW1i_Su_cJrCVOF2DMYeyYLOL_YVy6H21g7ur8mWF9eQgW4Clj4e8sxopnJNmQhSXwk9VKWLiEpwlgxBX8hS7bA2vOj75WT-0JxE-PTxCj9wmydwNeotfz1kaK-zWXuZ9szvmOM1zn7jI5PE9YdngHKHLguQqu6H1x8vbk4vLkxS9XJ5c7QUMv_l5HAOi6t-fTa9AbcfI3PCfJHHJLg7C3J-Db9zSqTKBp46TSzL4f2ZVHTvVBy9aNAWlwqQu8zPNmx00grpLbQ_MFTa7zbitQ7dt52TY5P3rQqUOFQ3bmMCFg3RlLpgZrq1PqU6t4xhPcetNCkt8nIMvfPeuoZJXwuMmV2ho7PzZ475aQfVT6r-qe8QnqivYIn7X6Xma7Lw82_EApd7aEp3chR39RRuOtTpMxVm1ehZqLVJ9km9vzezgnlak1O8WsBKTFdf8D_HSGkk6yNQt8LP6oEveY517UthgcU_OcfwsXuONUZFDY5GMWk1ECUPMkdkkgGJD7kvHrP1XywLAwtVD4eXukrC0Sl7-DIkd0xU8DYhPX7mmrwZua1qFFnQl6dDneplI2uIkXoYirXunZK3192WTKRVb7nfPjg2_RhOtoGA8YlkDAFF1KS6R31IxU2YFRhhmHV8Io692qhwXLnoIgZ_zw4qk8u2imWu2IYwlHsTB6X_VMXdkcT2i-wgOaeuPy4GbZFPodytOaemvtIOdzEdCepzk9sE4jbiWgZY4HAPBE7WpBAhpaaVIYA8vf5BaxVut0BaaDnTRkMKQxZfUzfetOnFHm536zFWWZDX4Ogp3LFbbo6h0bj97n-wIZiQaOjud68ToZhPk-AWE-PX_95ur0_PjkHfsTn1_Nfjo5B__o5MfZxS8o8DZ7dfJfbw5f8T4_MLgni4wCq9HgI552YIYPgyl2vw27LAgYKpZnJ-XdIPsc1qvLk8Lzfas0O3tzecVPCy_x0HMO9khdJuGwlUhbAue3h6_enBTvXfzf69Ojn4onP16cwIQurl4ens8uxEzah-f5dHaVTplob8FApxo4m66tq9Dz09clgsVvCi7w14vZ7NXh-XEzbuBbg_ADowM7LR4sVygKCBvL0GIrkrOzs9Ors5Nzhtbl6X-f1NBjD96c7YhccQNSiQuLcYEzOSus0bvYDb0Xp-cuYFJ0svF_E7m--NMBe1S8_vn06uXpeXeUz2cCbWJFm-Q65ofrMV7kuzfbkD08m705LyMJu_htK_UOk_K-HOZJchLFkRUzRYGYsUclyNkVSp4l_gEMvNPLYg7Q3mZlCR8ImFbcXFWMxSRV6gCzXPI7EVgbH5YpuLGWIHYFBPwDVE-WAgdmGy6ZOWIKvR9KmMWM7hCzzSrEs1r75ZxRd1G85QqdjuygDH4tbhbQsojIxPoWb8_KBD9xVfi01qEaz2zrJe8g-1ZsYWkNGQkocG5W3KT1TVHFCIogUyFVveA0XZihPi1lvdB6AsoZ-UgtQc6KnfApLiS3W0ixjAcVlr1fQqiob-Wxrf0lefHtycXpD7-UrywNGTATnCUkKVUHgLNaLKQG71ZD8od9DQIaMs6_EkTdJSgBHf6id5YwrSxm91CghcNYKrOiNeVXXoR_QzbM74uoXyeG3VWPiPYj3UwYJVc8vHr3anb009Xp2UkHCE3GRtEEswt9GRW5FvRgVJPuQO3tBT8XXxgCh7PHgc5I-3o_Ng_GMUel1lJeOrUnjZyjMY_A6WeqXIQH2gH0EqyWI64ckf5JGtU6FilYsLQQHPWFUyFVjOK31CfE1nz25gompVlgc8PukwFp4O45yygzW4CI5GlqIaLIxbU-K_KAk7XoLV0L3w7kVvqe6EM24F_xYJTXhSdT-wOsHv4YVJ7VILw6ubxETqy9qPKceAyoMGoaWES0cLZxRmlx6Jo-QetQRDrczJR6xEBDMCPiJpnWgKrST9z04yah4Lc6qXX1q_SVwuRyIcqYGaMkRLfKeSSR3G1Ewn9GbE5K7kGFqkYe1jfGHkrYY30rMqi_2h_2WUD31NjlOkWdnir0Kns6H57uNy88n7iBW0WjNoZWaaNrTeXt5ZsXxueDR_Wq62J8utOCHHXURnUqtWgklVq784iOklzMzvJSxxS3Xo5mx-b1mqo6phFZfHd-XPFQmR64o0_QVMUsRAxTfiOr9MTStdYB3Cp2XZV-FkkzlnU5hn7HBTjLcJapgwVVhF-iitbooNZxTzQECWMWQTW4_97c2hGtqSWKEviLHU30VrVSQ-AtXcfRA-cVbeR2u-oY49rPtaoCGfAs962BvAvNQIFG7Sr1gLRqW2S6hsCZEtazmAbcTpaSxZRM3TFJrVKdSgcGuaLFjnegVxd3ZQeaMqe9iGF-m1mZ9YM-ocY2O3ldKCFX78CJzEAkQZoLNjH7XNh8gX6msGIyPiEszSXuqWvyCNoZjcna2ey4mGY9MhAtTa6UDHcZdREXZo3RIeRI9UFzP8lq66NMi10egjf73aQYCtwArOUGAnN4hJ5mpGpOpSs4DMzJ8smC-QyCoOLW4MXDflUZin5HKq1KNhIaYrsbxLQGmOHfxfzVUGVBR0YgwJTXgsmdAUN_WARNFeI9kmt6m9Z4RLS_EqkToDy7AB11GJCUXUjGrBmQGwEUybdKf7aSdL1O15nZZ9hCyFbr7FRldEcrXacyF0SYIEw5ScEVXz8Im6zc-8hzfErCUenNYnxu9c32vEMcV0Tv21Rtc8PdhHW71yBanqH1cqVa6gsF_JfyEvtS83xO8NDRxOG_1qhG7JdYxX4JUyCcW_FKUXAQ0022MLiIbJet2GnARX9bXtmf4W2kJdAGxlaWtWvIgH8MW6Na0e0LuPZ8n4b4YDxxg4Us1hD2PfD8AxhPWf1jfaPtAbSAQiJxcCBBWQ796wIiOlQH0xPnraM0Ily9EtHg9zKPHBzlJSVJJoS04pPvEDoXlbhmPU0WwYZ9FmJBo_wqRe_LuJTaendI4zVGfNqg5JbEC7w49oILdUfm81RkhB4eVJ5tGxksf57TkA_cIhnArczLOHxXS_Acn77tyMW8vsSy7QNzvlA0E853kip7azqizMaRj1U3S_QvDvazQZ7IDW-TQtfmuLP0Py4_8HWpAzXQULbDpFtV6E1-b5yzlX6ttq5KyXbBcZoUqgwuXWj8OMfsM9WTgCL52TlQv2jS0PgkAUc4KON4dtkj263iSVteeZtkcZbTJNiaCGnmU01HPSqzJwRQuST8G_6pluVyk7N7vas7zc14dvStDY1Ne9NY3mME8uhJVvie71tnlp-KGxDh_Z_MZjEJRajEt8BEwUSsfSunY-atg7h0b-5WmAT_nFQkpqgq_VJklPyPd9UjHX2Ceb-0CF8I_6jBZ6X3agHE9im0iq17oG6adpzvzzcgFbcQJgmjckcwO7VY4E380u6wPdZyW7oCweehX41XgFzXvDqluH4TC8CCeEWazFZHO1U1b23WCSx8J64yr5Cj6p7qNtBvZQ0_W125VQFrU1c76bYvrZakRjLzQ20eX1SvPHoukr-R67fh3a4A2h2ux7oBccR3AHnRiZaKk5tNlu097RtB3FGxjyxciFucc0zZ3iHzLxrTWFgOY21WZoqwOlb5y7bMiP6k_Kv8TRuvBt_8Cx95xrbbZf2pPJzWqQJVNq4cf3p0yeYFxW8ymGs2a19t2K16UtYf8uog-e0EQ_GhplvZu0r6o6F9RVXMtHo2xdMpU7czpRSrBWRzjdysVh_3GJBFPYpUGCzCvJi9eiVddPwpy9m6jSCL0Wa1fHW1v2BpTG2KAioembDPHKLnxT45s2H3W8sv4xBRvJWnRee76p1v8doquGy_EJvjN4zxj16eHP0ky2i4PtF9zwLuSWIoisIdAfnpO7GBhae_ilGqpVBVKqgarBipaRdQLU1r3u7rq35GvUe55Vf9vmSBobrZx7RE8aYV7YoFKAJTNo6wZ2IoiNPl66J7oZ_Z1zJLugIBZAVf2OdZXLm7JBaBV_9lBSRok5UfTyo9A6yWs7IU1XiYJk9yK6H8Ix68tMdUVQvQ0Frs66vXsKU2a9qhNBqsJrrxo2QWP6tm_fWvhbutvaiQzUwRy9nL0z0HVmC1YifOzZOouAL1aWhKwq0Fae3Tq1eFcTPL85KYli4Pu4hXCifWstKzhlofhSCVdcVgG4cRsXeKxYD9kqq8pO8J_-4W5ac_BCjxJa04F3Wo4ktYayukQYwumrr9BSb3lrkSbMCyrgrgXJ7--PLw8uV8dn6CX6oCzsONUC62V-9EbSGfOqwWb0Hv4wzQBHZFxlAV3IZXMVU-rGodpXHyn_hxnKt3hZ6r8hjbcBdeTiGfwGJ2qXoRJ56GZY1ZZEJAhQLZmPzgphPuNgUxLiYWtjJa82hG57BGx3nWtMlWxReXDSsCsP4K97lwK01Elxw9lQYBvkWlb8JC9zVNAaYhnpSZab2Iuc9e8CMPiCB-BYjyT53Jb0Dg2Yd91SczuFgNxzhr3haqvhDalP5WQ0_F9eL2jY2GjOeMbH6YynQ-3iob4QmqKhWkjQQe1r4PWVzusdU9rHfRnUT1Y7_lh5T4Z38tsQw_lt9k-yFdL8Uxxj1x8LE41bjH8s1Mrr4dQKieod0Bmd1_WvSYyfoOrcuLo7MLC0-4Q0QAIgR2J17FVDk0uscyf4hieRhGPHcPmJd0cvXm4nx_f1959avtHVgnSqllCfiDmFj9hJDpk1uPOfOzg0eMJY7sMzDFSbjX7E8Fm8r3YqQbXWT5aycX-IEFvZCjVTEYlYL01fHj22vxCW7mAohvcJUHDUA_6WLKnWK5mSF813rp6tmbV0XKYov3r6Qvqh4rr6pgn1x0rAQ_Zs6mr9V47GbPpO655ZpQ--h5cUQjifPy2Ln1rVisp_XjTD8LV1miyJzBPv8YObAm87NX6_Q2DkvrR5N0c33DLTSSnLmMH4UjVwqqONdh8B3bJ8j1fVXjSz57on4HM41K11fJ_PxF0NcufnOK39y6EZidnl9dvUNV12YgtIzpzJiwrDYxpDREE-YqiYP8orGWsZfcqe3uMXV7fHh1CCrXaBxPEyDBGnxU7uBUSSjWMTSdgZXOuOorW5U7yPAcoMyEiErjZ99ZqNU9V9QT8KcQyRIiYtmxOgOkMcxZEgHaTcb6BQffPWM_EcLUGZghgEFqgdC8NDUuK0nDbkBBTSFJ1HBMWfSSkcsduBrAkBQc71D5gqvxjHEzuWaVM2KCShpBTG2Y9RYANCJpJbDMIy_TRAIFq4KDAKPnIRp2XARKlT5itQx9Wh06ZTtYt7WKCgA9XxhSruX5novcn7S3ya3TgBnei0KeFwiW0bmeV2veGW4L26WiLiu-CpNbPa-nVyxsG1FTCqaRZ1oqsBIRVrc1LfFlYfEVYb9UtcoKKJuciqtTvOYV3rf81HZdCPXzrk0191UdqG1OGGhiajMyApi17zBoRDpSQn7jBnp9els3LFGbt5kdo4TMilSU26S8SlyX7CA98hS70EO1_s0Y18N7uxiubWO9GDlJrR-MWtd42rRFTnbnTKMOqTncrQxq3szagmkxcebHmWDtEtY-Yur9oiemHu6Kb4BjMgIj4ZysRc1bQYpt_khH7VhLqEiyV2IZVXFnYlMlS5eUnxLC2qr_6KILzKK8gy6oKqCqStCUTdMpyIoxKDz6stAbTxnL2ePXxrEkiXkB9D5fEw2jhlrKmcinN5VpNLCd6Oq1sl4r-9UsD-dH1cHH5WIsqB4M1A_Srit5PK8oNu04ea-bHZuZtqXKNMq23SyeYtGuHNwtsVL2-F0SKpXbhjpgWemhJlO-Y_cFzjQP_PWby5fsEru558pL-ZRLdVRH9HuJa3FLGf_B7iBDeS-BC-e8BA7GsBE4J1cT8Mp-40500Po8lhLyPqLfiBZV8O3U4MnBhpuBOlCloedjaFO_veg3oFDzIDvRqbyJSbmt0_RWIURxkaXhm-o19AwfuK8xvLzxSOtZ-wr280LPldMRP4o3xe1_6iTr39OuT7XepnJxHEv_8Y1MKj9nLNvOS7-ER7g5Xa6wdpinvMsIByzF0vr-iej25DnP7YnI7fsK1OfVhWpYIRXTNrLrWtN8b345wraWff0y_i63YbG767PNapWuGb_92gMTCp7VyRH7fi6__xS_UvW_p-sagg==)):

```
    "dao_config_data": {
      "data": {
        "bytecode": {
          "dao_config_dust_limit": "800",
          "dao_config_fee_allowance": "1600",
          "dao_config_safebox_nominal_value": "1000000",
          "dao_config_rewards_minimum_increment": "1000000",
          "dao_config_executor_fee": "1000000",
          "dao_config_max_safeboxes": "2000",
          "dao_config_vault_close_time": "1684430000",
          "dao_config_safebox_open_time": "1684431000",
          "dao_config_rewards_remainder_beneficiary": "0x76a914586f33d3cd763ec12ea1829ad834c6e3d345131b88ac"
        }
      },
      "description": "DAO configuration data, required to build Vault and Safebox locking bytecodes.",
      "name": ""
    },
    "dao_genesis_blueprint_data": {
      "data": {
        "bytecode": {
          "genesis_executor_fee": "500000",
          "genesis_bcmr_output_bytecode": "0x76a914e11bc0b45edb8c7caca8b63f4d02eca619dbc23088ac",
          "genesis_message": "'Happy Bitcoin Cash May 2023 upgrade day!'",
          "genesis_designer_pubkey": "designer_key.public_key"
        }
      },
      "description": "DAO genesis blueprint data, required to build Blueprint locking bytecode.",
      "extends": "dao_config_data",
      "name": ""
    },
```

DAO "Vault" transactions:

- Genesis transaction: https://chipnet.imaginary.cash/tx/0df6cfd62fdc314fd40c07425296372069e1b96606439e3096925615449dea68
- Mint No. 1 - locking up exactly the nominal amount + bonus: https://chipnet.imaginary.cash/tx/093bf4536a89fb8d340d4cc0099e9bd1431d42d6d9722cfd635a8a498f40c763
- Mint No. 2 - same: https://chipnet.imaginary.cash/tx/6afeb2a173e8e8b612653307c2a263415aa31df08f29866a0fb350a987f956cf
- Mint No. 3 - user adds his own extra to the safebox: https://chipnet.imaginary.cash/tx/dc6ca48ee3e3a43b0d84ae51b2eeb8b25abcaa7ca37b943fde1c62ff7e1344fa
- Mint No. 4 - user gives up the bonus (vault amount not changed) and locks up exactly the nominal amount: https://chipnet.imaginary.cash/tx/dc6ca48ee3e3a43b0d84ae51b2eeb8b25abcaa7ca37b943fde1c62ff7e1344fa
- Mint No. 5 - user locks exactly the nominal amount, while adding a little to DAO's reward pool: https://chipnet.imaginary.cash/tx/dc6ca48ee3e3a43b0d84ae51b2eeb8b25abcaa7ca37b943fde1c62ff7e1344fa

User P2PKH transactions:
- User transfers No. 2 (p2pkh): https://chipnet.imaginary.cash/tx/716f5ccd31fdd67390dd78b3c04d4d3d180a67c36df772ab16ed263b2847753d

DAO "Safebox" transactions:
- Withdraw No. 2 - user burns his No. 2 keycard NFT with the matching "safebox" NFT to withdraw from the DAO: https://chipnet.imaginary.cash/tx/f8c34d51bbaf8109de3d543986a71032fa4f88b6329aca7f731d34c79bd77410

bitjson commented:
Thanks for your patience @A60AB5450353F40E. Nice work getting the website up, and thanks for the chipnet demo! 

There are just a few other issues we need to cover. I just pushed up a new pull request template to help streamline new listings – could you try it (esp. the checklist) and see what you think? You can copy the raw markdown directly into a comment here and the checkboxes will become functional. https://raw.githubusercontent.com/OpenTokenRegistry/otr.cash/b07c9d110566406d9122acfeff5fa45590357f6b/.github/pull_request_template.md

A60AB5450353F40E commented:
<!-- For new listings and metadata updates, please fill in the template below. For other pull requests, please delete this template and describe your pull request. -->

# New Listing: `EMRDAO-23U` <!-- or `NAME` for non-token identities -->

<!-- Replace the `PROVIDE...` fields below to add context to this PR: -->

- Identity Name: `Emerald DAO 23U`
- Authbase: [`180f0db4465c2af5ef9363f46bacde732fa6ffb3bfe65844452078085b2e7c93`](https://blockchair.com/bitcoin-cash/transaction/180f0db4465c2af5ef9363f46bacde732fa6ffb3bfe65844452078085b2e7c93)

## Evidence Linking Authbase to Identity

<!-- First, we need to prove that the on-chain identity defined by the authbase self-identifies as the submitted identity. This prevents an attacker from publicly claiming to control an authbase they do not control. Mark the type of proof you're providing by entering an `x` in the appropriate checkbox: (Change one `[ ]` to `[x]`) -->

- **Select one**:
  - [x] **Direct evidence**: a [chain-resolved registry](https://cashtokens.org/docs/bcmr/chip#chain-resolved-registries) at the authhead provides substantially equivalent metadata for this authbase. The Authhead TXID is provided below:
  - [ ] **Indirect evidence**: the identity is [burned](https://cashtokens.org/docs/bcmr/chip#burned-identities), so the authbase can safely be linked using indirect evidence. Two or more corroborating, third party-archived URLs of reputable sources vouching for this association are provided below:

<!-- If "Direct evidence" is checked, provide the Authhead TXID here.

If "Indirect evidence" is checked, provide at least two third-party archived URLs (archive.org, archive.is, etc.) of reputable sources vouching for this association. If the archiving service does not include the original URL in the archived URLs (i.e. it also shortens URLs), please also provide the original URL. -->

## Evidence Linking Identity to Authbase

<!-- Next, we need to prove that the real-world entity self-identifies as the on-chain identity defined by the authbase. This protects known entities from being associated with an attacker-controlled on-chain identity. -->

Archived URL(s):
- [Current authead](https://blockchair.com/bitcoin-cash/transaction/d756d3c80e08ba9f6f3a8417d21d519b15a4a3e342775f9fe4c3b8da856ca75a)
- https://web.archive.org/web/20230521235129/https://emerald-dao.cash/#deployments

<!-- Provide a third-party archived URL to a statement by the represented identity claiming the submitted authbase as the root of their on-chain identity. Multiple source are encouraged (e.g. a website, social media profiles, etc.) -->

## Evidence Linking Authbase to Categories

<!-- If they differ, we need to prove that all submitted token categories are/were issued or controlled by the authbase.

As with linking the authbase to this identity, we'll need to demonstrate that each claimed category self-identifies as associated with the authbase via either direct or indirect evidence. -->

- **Select one**:
  - [x] Token `category` is equal to the authbase for all [Identity Snapshots](https://cashtokens.org/docs/bcmr/chip#identity-snapshots).
  - [ ] Evidence is provided for differing `category` values below:

<!-- Provide evidence for each category claimed in the history of the identity other than the authbase. Uncomment and duplicate the below section as needed: -->

<!--

- Differing `category`: `PROVIDE_CATEGORY_ID`
- **Select one**:
  - [ ] **Direct evidence** – category's authhead TXID with a corroborating chain-resolved registry:
  - [ ] **Indirect evidence** – the category's identity is [burned](https://cashtokens.org/docs/bcmr/chip#burned-identities) or widely understood to be lost, abandoned, compromised, or destroyed; two or more third-party archived URLs of reputable sources vouching for this association:

[Provide evidence here]

-->

## Substantiation of Other Claims

<!-- Finally, if the submitted `description` makes any claims about the behavior of tokens, please submit any additional information to help reviewers corroborate statements of fact in your `description` or other fields. -->

- **Select one**:
  - [ ] The submission (esp. the `description`) includes no claims requiring substantiation.
  - [x] The claims in this submission can be corroborated with the following evidence:

<!-- Please provide third-party archived URLs to security reviews, attestations, audits, and other resources that substantiate this submission. For each resource, please provide an summary of the claims that can be verified using the resource. -->

- Contract code: https://gitlab.com/0353F40E/emerald-dao/-/tree/main/contracts/v2-1-0
- This contract instance particulars: https://gitlab.com/0353F40E/emerald-dao/-/tree/main/instances/v2-1-0/180f0db4465c2af5ef9363f46bacde732fa6ffb3bfe65844452078085b2e7c93
- On-chain tests ("chipnet"): https://github.com/OpenTokenRegistry/otr.cash/pull/1#issuecomment-1553463521

# Checklist

- [x] I have reviewed the [OpenTokenRegistry inclusion criteria](https://otr.cash/docs/list#inclusion-criteria), and this listing can be included in the registry.
- [x] The listing completes all required fields:
  - **`name`**
    - [x] Objective, authoritative, and neutral tone
    - [x] No longer than 20 characters; exceptions may be allowed for existing legal names (please provide evidence above including the jurisdiction(s) in which the legal name is established).
  - **`description`**
    - [x] Objective, authoritative, and neutral tone
    - [x] Descriptions is appropriate for use in user interfaces without a disclaimer by unaffiliated, neutral, third parties (wallets, block explorers, exchanges, etc.); no exclamations, calls to action, or disputable statements.
    - [x] All claims are substantiated above.
  - **`token` information** (if applicable)
    - [x] Token `category` is defined.
    - [x] If fungible tokens exist for the `category`, `decimals` is defined.
    - **`symbol`**
      - [x] Globally-unique base symbol, including among assets not listed in OpenTokenRegistry.
      - [x] Valid symbol (regular expression: `/^[A-Z0-9]+[-A-Z0-9]*$/`)
      - [x] Appropriate minimum length (for new assets: 4 or more characters for primarily fungible base symbols, 6 or more characters for primarily non-fungible base symbols)
      - [x] No longer than 13 characters for base symbols
      - [x] No longer than 26 characters for full symbols
  - **`uris.icon`**
    - [x] Uses IPFS (`ipfs://`)
    - [x] Either `SVG` format OR 400px by 400px `AVIF`, `WebP`, or `PNG`
  - **`uris.web`**
    - [ ] Domain name is owned by the issuing project or organization
    - [x] Either `HTTPS` protocol OR ongoing activity demonstrated via optional URI identifiers (described in [Substantiation of Other Claims](#substantiation-of-other-claims))
  - [x] Includes relevant historical information in previous snapshots OR this token has never been rebranded, redenominated, or reissued
  - [x] All static data (images, animations, videos, binary files, etc.) uses `IPFS` to ensure file integrity (`ipfs://...`).

A60AB5450353F40E committed:
Update bitcoin-cash-metadata-registry.json

A60AB5450353F40E committed:
update ticker to EMRDAO-U23

A60AB5450353F40E committed:
Update description

bitjson reviewed (approved):
Thanks for your work and patience in helping to iron out our listing process @A60AB5450353F40E! This listing is looking good, are you ready for it to be merged?

A60AB5450353F40E commented:
yes, all good!

bitjson commented:
Squashed and merged, thanks @A60AB5450353F40E! 

Would you be willing to be added to @OpenTokenRegistry/reviewers? We'll require approvals from multiple known reviewers for future PRs, so it's great to add known contributors who are familiar with the process/requirements!

A60AB5450353F40E commented:
>Would you be willing to be added to @OpenTokenRegistry/reviewers?

Sure!

Btw, a little error snuck in during our review/edit, the linebreaks have the wrong slash, `/n` should be changed to `\n`
